docs/acme-divergences.md

```diff
@@ -18,8 +17,6 @@ For all rate-limits, Boulder includes a `Link` header to additional documentatio
 
 Boulder uses `invalidEmail` in place of the error `invalidContact`.
 
-Boulder does not implement the `unsupportedContact` and `dnssec` errors.
```

Comment:
Nothing to address here. It's interesting that we flagged `dnssec` here when it's not classified as an error type outside of simply `dns`.

Reply:
Right? I imagine this is one of the errors that existed in an earlier draft of the spec.